### PR TITLE
March 2024 evaluation updates

### PR DESCRIPTION
**Pull request by nickswalker**
This was the state of the code on the control machine as run during the last evaluation. We should merge this before the branch starts accumulating additional changes

### CODE REVIEW

File: aurmr_tasks/src/aurmr_tasks/common/motion.py [hunk 1]
```diff
@@ -313,12 +313,43 @@ def execute(self, ud):
...
+            constraints.joint_constraints.append(new_constraint)
+        print(constraints)
+        success = self.robot.move_to_joint_angles(joint_angles, path_constraints=constraints)
+        if success:
+            return "succeeded"
+        else:
+            return "aborted"
+        
 class MoveEndEffectorToPose(State):
-    def __init__(self, robot, default_pose=None):
```
[the remark this answers is not in the document]
Comment (markusgrotz) [reply aurmr_tasks/src/aurmr_tasks/common/motion.py:318]: Readded

File: aurmr_vacuum_gripper/vacuum_gripper_control/nodes/VacuumGripperSocketNode.py
Comment (markusgrotz) [inline aurmr_vacuum_gripper/vacuum_gripper_control/nodes/VacuumGripperSocketNode.py:1]: These changes shouldn't be added

**Review by markusgrotz — CHANGES_REQUESTED** [review]

**Comment by markusgrotz** [comment]
Can we revert https://github.com/au-rmr/aurmr_tahoma/pull/31/commits/b5b7922

**Comment by nickswalker** [comment]
Yes, will change in the morning

**Review by markusgrotz — APPROVED** [review]

**Pull request merged**